packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -668,6 +679,60 @@ class Auth {
         return await this.loginGuestUser()
     }
 
+    /**
+     * A wrapper method for commerce-sdk-isomorphic helper: authorizeIDP.
+     *
+     */
+    async authorizeIDP(parameters: Parameters<Helpers['authorizeIDP']>[1]) {
+        const privateClient = !!this.clientSecret
+        const redirectURI = this.redirectURI
+        const usid = this.get('usid')
+        const {url, codeVerifier} = await helpers.authorizeIDP(
+            this.client,
+            {
+                redirectURI,
+                hint: parameters.hint,
+                ...(usid && {usid})
+            },
+            privateClient
+        )
+        window.location.assign(url)
+        this.set('code_verifier', codeVerifier)
+    }
+
+    /**
+     * A wrapper method for commerce-sdk-isomorphic helper: loginIDPUser.
+     *
+     */
+    async loginIDPUser(parameters: Parameters<Helpers['loginIDPUser']>[2]) {
+        const codeVerifier = this.get('code_verifier')
+        const code = parameters.code
+        const usid = parameters.usid
+        const redirectURI = parameters.redirectURI || this.redirectURI
+
+        const token = await helpers.loginIDPUser(
+            this.client,
+            {
+                codeVerifier,
+                clientSecret: this.clientSecret
+            },
+            {
+                redirectURI,
+                code,
+                ...(usid && {usid})
+            }
+        )
+        // Should this be false ?
+        const isGuest = false
```

Comment:
Wanted to confirm this with the PWA team!👀

Reply:
I do believe you are right. You can't login via IDP and be a guest user, so it makes sense that this is false.

Reply:
ideally it would always be nice to decode the token and grab the flag from the token but in this case I think it's safe to assume the user is a registered user.